pkgs/tools/misc/zoxide/default.nix

```diff
@@ -15,15 +24,32 @@ rustPlatform.buildRustPackage {
     owner = "ajeetdsouza";
     repo = "zoxide";
     rev = "v${version}";
-    sha256 = "1ghdal6pqkp56rqawhj26ch1x4cvnjj032xz3626aiddqgn134zj";
+    sha256 = "143lh94mw31pm9q7ib63h2k842g3h222mdabhf25hpb19lka2w5y";
   };
 
+  # tests are broken on darwin
+  doCheck = !stdenv.isDarwin;
+
+  # fish needs a writable HOME for whatever reason
+  preCheck = ''
+    export HOME=$(mktemp -d)
```

Comment:
nitpick
```suggestion
    export HOME=$TMPDIR
```

Reply:
I see both being used in Nixpkgs. Why would `$TMPDIR` be used over `mktemp -d`?

Reply:
There aren't any substancial difference, I just prefer using $TMPDIR since it's already there. 
Sorry for the noise.

Reply:
It's all good. I was just wondering if I've been doing it wrong because I "learned" to use `mktemp` rather than `TMPDIR`.